google/cloud/grafeas/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+# ~~~
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+include(GoogleapisConfig)
+set(DOXYGEN_PROJECT_NAME "GrafeasPub/Sub Lite API C++ Client")
+set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the Pub/Sub Lite API")
```

Comment:
pub/sub lite?

Reply:
How else could you know where I cut&pasted this file from?